front-end-master/about_us/about.js

```diff
@@ -1,8 +1,8 @@
-class Division{
-	 constructor{
-
-	 }
-}
+//class Division{
```

Comment:
Remove the commented section if not needed.

Reply:
I actually need this but not now. I added this so that we can change the content of the people's bio and other stuff dynamically. i will comment out the whole person section section too as we don't need it right now. or should i make another file for the class declarations and add that to gitignore?

Reply:
Why would we put it in the gitignore though? If you need this, I'd keep it here and address it in the next branch.